src/riak_kv_pncounter.erl

```diff
@@ -50,6 +50,15 @@
 -export_type([pncounter/0, pncounter_op/0]).
 
 -opaque pncounter() :: {riak_kv_gcounter:gcounter(), riak_kv_gcounter:gcounter()}.
+%% Redeclaring pncounter() with -type to please dialyzer, which would
+%% otherwise issue the following warning:
+%%  riak_kv_pncounter.erl:100: Invalid type specification for
+%%    function riak_kv_pncounter:merge/2. The success typing is
+%%    ({[{_,pos_integer()}],[{_,pos_integer()}]},{[{_,pos_integer()}],[{_,pos_integer()}]})
+%%    -> {[{_,pos_integer()}],[{_,pos_integer()}]}
```

Comment:
I noticed dialyzer tends to issue "summary" warnings, such as "no local return", which are consequential to some other issue with the function in question, and which disappear when that underlying issue is fixed.

Anyway, I think I should just trim the hairiness of this comment (in a follow-up commit).